pyproject.toml

```diff
@@ -81,8 +81,9 @@ test = [
     "coverage",
     "pytest>=6.0.0",
     "pytest-cov",
-    "pytest-random-order",
+    "pytest-randomly",
```

Comment:
Just curious, what does this do differently from `pytest-random-order` that improves the speed?

I read this: https://pytest-dev.github.io/pytest-order/stable/other_plugins.html, but it doesn't seem to have an explanation for the speed improvement.

Reply:
ah no xdist is the pytest plugin that parallelize but pytest-random-order is not compatible with it so I had to pick another plugin to randomize the test order